### ISSUE TITLE
Suggestion: Use a single deployment location.

### ISSUE THREAD

**Issue by antis81** [almindor/lighthouse#25]
Like probably most Sailfish users, I have "Storeman" installed in parallel with the "Jolla Store. This got me a little confused, because I had installed lighthouse from Storeman - thus the OpenRepos.net package, which seems quite old (1.3.x something).

1. Storeman shows 1.6.5 as the current version, but it actually installs the 1.3.x version saying it is up-to-date. It is likely never upgrading.
2. Installing from Jolla store installs the 1.6.4 version - 1.6.5 not available yet.  

Thus, my little fix from yesterday is actually based on a false assumption, because rotation kind of worked in the 1.6.4, but not in 1.3.x). Anyway the fix is a fix is a fix… it still does the right thing. :smile: 

Ideally both package managers would install from the exact same location. Is there any reason to split it up?

**Comment by antis81** [comment]
A little update:
1. Uninstalled Storeman version
2. Installed Jolla Harbor version (1.6.4)

In this state Storeman marks lighthouse as "uninstalled" and the available version as 1.5.0 - actually matching the oldest RPM package version available on [OpenRepos.net](https://openrepos.net/content/almindor/lighthouse).

And Jolla Harbor? Tada… after some time Jolla Harbor notified me about an upgrade 1.6.4 -> 1.6.5. Finally!

**Comment by almindor** [comment]
I'm not sure I can do anything about this except maybe remove lighthouse from openrepos. I forgot to update on openrepos for a little while. I don't know exactly how storeman works. My usual release process is to go to both harbour and openrepos and push there.
